scripts/languageserver/provider_misc.jl

```diff
@@ -75,10 +75,36 @@ end
 abstract didChange <: Method
 
 function Respond(r::Request{didChange,DidChangeTextDocumentParams})
+        vtdi = r.params.textDocument
+        C = r.params.contentChanges
+        for c in C
+            c(vtdi)
+        end
     try
-        documents[r.params.textDocument.uri] = split(r.params.contentChanges[1].text, r"\r\n?|\n")
         return
     catch err
         return Response{didChange,Exception}("2.0",r.id,err)
     end
 end
+
+function (c::TextDocumentContentChangeEvent)(vtdi::VersionedTextDocumentIdentifier)
+    d = join(documents[vtdi.uri],'\n').data
```

Comment:
Actually, looking at it the type specification on the document Dict seems to force a conversion when we open a file